system_v10/src/lib.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019-2022 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+use cid::Cid;
+use fvm_ipld_encoding::tuple::*;
+use fvm_shared::METHOD_CONSTRUCTOR;
+use num_derive::FromPrimitive;
+
+/// System actor methods.
+#[derive(FromPrimitive)]
+#[repr(u64)]
+pub enum Method {
+    Constructor = METHOD_CONSTRUCTOR,
+}
+
+/// System actor state.
+#[derive(Default, Deserialize_tuple, Serialize_tuple, Debug, Clone)]
+pub struct State {
+    // builtin actor registry: Vec<(String, Cid)>
```

Comment:
remove this?

Reply:
Why? Corresponding v9 actor has it

Reply:
Ah i see, it was already there in the orignal source.